### DISCUSSION TITLE
Options --> Lock is missing

### DISCUSSION THREAD

**Discussion by cyb3rz3us** [standardnotes/forum#2558] [General] [answered]
I used to be able to lock a given note by clicking on Options --> Lock.  I no longer see this feature - was it removed?

**Additional context**
Using app v3.8.6; issues occurs with all editors.

**Comment by JaspalSuri** [comment, marked as answer]
Hi @cyb3rz3us, the `Lock` feature has been renamed to `Prevent Editing` as some users would confuse it for the `Protect` feature.
